scripts/mkcloud

```diff
@@ -134,6 +136,7 @@ fi
 pidfile=mkcloud.pid
 
 trap 'error_exit $? "error caught by trap"' TERM
+exec 3<&0
```

Comment:
even better would be to move the crowbar_dev_guard part into a separate script, because with the interactivity it is so different - also avoids having to make sure that no other mkcloud steps are run along with it.